.github/workflows/pythonapp.yml

```diff
@@ -223,6 +231,16 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          activate-environment: pdftotext
+      - name: Install pdftotext
+        run: |
+          conda install -c conda-forge poppler pdftotext
+      - name: Conda list
+        run: |
+          conda list
```

Comment:
I think this can be removed, it just prints a list of installed packages and was in my example purely to showcase that everything installs fine.
```suggestion
```